onnxruntime/core/framework/session_state.cc

```diff
@@ -588,15 +588,15 @@ Status SessionState::PrepackConstantInitializedTensors(
                   // within this session. Or if the weight is not present on disk,
                   // we store the newly minted pre-packed data.
 
-                  AllocatorPtr session_cpu_alloc = GetAllocator(kernel->Info().GetDevice(OrtMemType::OrtMemTypeDefault));
+                  AllocatorPtr session_initializer_alloc = GetInitializerAllocator(kernel->Info().GetDevice(OrtMemType::OrtMemTypeDefault));
```

Comment:
>AllocatorPtr session_initializer_allo [](http://example.com/codeflow?start=18&length=37)

Q. Is it guaranteed accessible from CPU code? We need to compute the container key. Sorry, I do not remember the properties of the an allocator.